Modules/Quotas/src/Quotas.ConsumerApi/Controllers/QuotasController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Backbone.BuildingBlocks.API;
+using Backbone.BuildingBlocks.API.Mvc;
+using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+using Backbone.BuildingBlocks.Application.Pagination;
+using Backbone.Modules.Quotas.Application;
+using Backbone.Modules.Quotas.Application.DTOs;
+using Backbone.Modules.Quotas.Application.Identities.Queries.GetQuotasForIdentity;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using ApplicationException = Backbone.BuildingBlocks.Application.Abstractions.Exceptions.ApplicationException;
+
+namespace Backbone.Modules.Quotas.ConsumerApi.Controllers;
+
+[Route("api/v1/[controller]")]
+[Authorize("OpenIddict.Validation.AspNetCore")]
+public class QuotasController : ApiControllerBase
+{
+    private readonly ApplicationOptions _options;
+
+    public QuotasController(IMediator mediator, IOptions<ApplicationOptions> options) : base(mediator)
+    {
+        _options = options.Value;
+    }
+
+    [HttpGet("ListIndividualQuotas")]
+    [ResponseCache(Duration = 1800, Location = ResponseCacheLocation.Any, VaryByQueryKeys = new string[] { "address" })]
```

Comment:
If I understand the ResponseCache correctly (and it's possible that I don't, since I never used it), then it is configured wrong here. I assume that this now leads to the same response for every user as long as the cache is valid. Because you set `VaryByQueryKeys` to `address`, but there is no such property in the query. So this means that the cache is the same for all requests.